### DISCUSSION TITLE
What estimator should I use?

### DISCUSSION THREAD

**Discussion by mreismendes** [awslabs/gluonts#992] [Q&A]
Hello!

I need some advice about the two main characteristics of my problem.

1) What is the more suitable estimator for my problem?

I have daily record data with one target and 28 other related variables. Sice, I don't know in advance the values of related variables, and knowing that each variable depends not only on its past values but also has some dependency on other variables, what estimator is more suitable for my problem? GPVAREstimator?

2) My data doesn't have weekend records, only weekdays records! What should I do related to the time-frequency?

Do I need to change the freq or should I set as "D"?

Thank you!

**Comment by timoschowski** [comment]
(1) you have two choices: either you choose a multi-variate model (e.g., https://gluon-ts.mxnet.io/api/gluonts/gluonts.model.deepvar.html as an alternative) or you pick a seq2seq model where you don't have to have the feature/co-variates in the forecast horizon: https://gluon-ts.mxnet.io/api/gluonts/gluonts.model.seq2seq.html

(2) you can do "daily" and then mark the values as missing/NaN during weekends and add a binary indicator co-variate to indicate it's weekend. But maybe there's a workday granularity as well, that I'm not aware of.

**Comment by mreismendes** [comment]
@timoschowski Thank you! 

In what kind of problem should I use GPVAREstimator instead of DeepVAREstimator?

**Comment by timoschowski** [comment]
Gaussian Process models are something that would require something failry smooth, I'd say -- and the implementation may not be as well-tuned, I think. In general, the multi-variate models aren't as far implementation-wise, so you'll have to adjust your expectations.

I'd suggest to try a seq2seq model first.

**Comment by mreismendes** [comment]
@timoschowski Thank you for your help. Do you have any example of end-to-end implementation of seq2seq models?

Thank you!
